### PR TITLE
Team 13's Tutorial

### PR DESCRIPTION
**Pull request by madelinefruin**
please approve it

### CODE REVIEW

File: content/tutorials/Team13Tutorial.md [hunk 1]
```diff
@@ -0,0 +1,237 @@
+---
+title: Battery Indicator with ESP32
+date: 2024-11-21
+authors:
+  - name: Andy Tu
+  - name: Madeline Fruin
+  - name: William Luong
+---
+
+<img src="Team13Photos\BatteryCharged.jpg" alt="[Circuit Fritz](https://www.vectorstock.com/royalty-free-vector/fully-charged-battery-color-icon-vector-28877489)"> </img>
```
Comment (AdinAck) [inline content/tutorials/Team13Tutorial.md:10]: this is not how you embed images in markdown. refer to the markdown spec listed in the instructions to learn how to do this.

File: content/tutorials/Team13Tutorial.md [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+Let's take a dive into the indvidual blocks of code. 
+
+<pre><code class = "language-arduino"> //Constants
+  const int batteryPin = 1;
+  const float referenceVoltage = 3.3;
+  const int resolution = 3950;
+  const float voltageDividerRatio = 2.0;
+  const float maxBatt = 4.2;
+  const float minBatt = 3;
+</code></pre>
```
Comment (AdinAck) [inline content/tutorials/Team13Tutorial.md:150]: this is not how you create code blocks either.

File: content/tutorials/Team13Tutorial.md [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+
+We would not need to change the voltage divider for a 4.2V battery because 4.2V/2 is 2.1V, which is under the maximum that the board can take in. We would only need to change the minBatt and maxBatt values in the code.
+
+We would need to change the voltage divider for a 12V battery because 12V/2 is 6V, which is nearly double what the board can take in! We could choose R1 = 300kΩ and R2 = 100kΩ to get 1/4 of 12V, or 3V. We would have to change our maxBatt, minBatt, *and VoltageDividerRatio* in the code.
+
+## Additional Resources
+
+### Useful links
+
+<a href="https://www.monolithicpower.com/learning/resources/an-introduction-to-batteries-components-parameters-types-and-chargers"><here>Website that tells the nominal voltage and voltage range of many batteries.</a>
```
Comment (AdinAck) [inline content/tutorials/Team13Tutorial.md:237]: this is not how you create links. you cannot use any HTML at all.

**Review by AdinAck — CHANGES_REQUESTED** [review]
i left some notes for some simple changes

File: content/tutorials/Team13Tutorial.md [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+```
+{
+  const int batteryPin = 1;
+  const float referenceVoltage = 3.3;
+  const int resolution = 3950;
+  const float voltageDividerRatio = 2.0;
+  const float maxBatt = 4.2;
+  const float minBatt = 3;
+}
+```
```
Comment (AdinAck) [inline content/tutorials/Team13Tutorial.md:151]: this looks like arduino code (which is C). you should mark the code blocks as C so syntax highlighting is enabled.

File: content/tutorials/Team13Tutorial.md [hunk 5]
```diff
@@ -0,0 +1,244 @@
...
+
+Give a brief explanation of the technical skills learned/needed
+in this challenge. There is no need to go into detail as a
+separation document should be prepared to explain more in depth
+about the technical skills
+
+If you've taken a look at the battery we gave you, you might notice that it says 3.7V on it. The maximum voltage input for the ESP32 is 3.3V, so we need to find a way to decrease the voltage from the battery, we can do this with a <b>voltage divider</b>.
+
+A voltage divider is a passive linear circuit made up of two resistors, although sometimes more are used to get a specific resistance value. 
+![Voltage Divider Schematic and Equation](Team13Photos/voltagedivider.png)
```
Comment (AdinAck) [inline content/tutorials/Team13Tutorial.md:94]: the ECE 196 website background is basically black, and this image is black strokes on a transparent background so it is not visible. here's what it looks like:

image

**Review by AdinAck — CHANGES_REQUESTED** [review]
just a couple more things

**Comment by madelinefruin** [comment]
Changing the png to have a white background lessened the quality a little :(

**Comment by AdinAck** [comment]
> Changing the png to have a white background lessened the quality a little :(

That's ok it looks great!

This tutorial looks good! I'll merge it now.

Good work!

**Comment by madelinefruin** [comment]
Slay, thank you!

**Review by AdinAck — APPROVED** [review]

**Comment by madelinefruin** [comment]
Btw, you might want to make an announcement about making the fork here thing because 4 or 5 other groups forked from where we originally had everything. And as of right now, I can't see any other forks from docs.

**Comment by AdinAck** [comment]
> Btw, you might want to make an announcement about making the fork here thing because 4 or 5 other groups forked from where we originally had everything. And as of right now, I can't see any other forks from docs.

Yes I updated the instructions to directly link to the docs repo minutes after you showed me the ambiguity. Thanks!

**Comment by AdinAck** [comment]
[here](https://ece-196.github.io/docs/tutorials/team13tutorial/) is your tutorial on the web! congrats on being the first!

**Pull request merged**